tests/setup.ts

```diff
@@ -6,7 +6,15 @@ export interface PluginWithPrivates extends PomodoroPlugin {
   _timer: PomodoroTimer;
 }
 
-jest.mock('obsidian'); // This will use the __mocks__/obsidian.ts automatically
+jest.mock('obsidian', () => {
+  const obsidian = jest.requireActual('obsidian');
+  const moment = jest.requireActual('moment');
+
+  return {
+    ...obsidian,
+    moment
+  }
+}); // This will use the __mocks__/obsidian.ts automatically
```

Comment:
This was required for `moment` to work with the tests.  
It might be a reason why the Settings mock breaks and we get an error with `.addDropdown()`